ARCs/arc-0008.md

```diff
@@ -46,7 +46,7 @@ In case the wallet uses an API service that is secret or provided by the user, t
 
 > Leakage may happen by accidentally including too much information in responses or errors returned by the various methods. For example, if the nodeJS superagent library is used without filtering errors and responses, errors and responses may include the request object, which includes the potentially secret API service URL / secret token headers.
 
-For dApps using the `signAndPostTxns` function, it is **RECOMMENDED** to display a Waiting Screen until the transaction is confirmed to prevent potential issues.
+For dApps using the `signAndPostTxns` function, it is **RECOMMENDED** to display a Waiting/Loading Screen to wait until the transaction is confirmed to prevent potential issues.
```

Comment:
```suggestion
For dApps using the `signAndPostTxns` function, it is **RECOMMENDED** to display a Waiting/Loading Screen to wait until the transaction is confirmed to prevent potential issues.

> The reasoning is the following: the pop-up/window in which the wallet is showing the waiting/loading screen may disappear in some cases (e.g., if the user clicks away from it). If it disappears, the user may be tempted to perform again the action, causing significant damages.
```